lib/stoplight/default.rb

```diff
@@ -2,16 +2,15 @@
 
 module Stoplight
   module Default
-    WHITELISTED_ERRORS = [
-      NoMemoryError,
-      ScriptError,
-      SecurityError,
-      SignalException,
-      SystemExit,
-      SystemStackError
-    ].freeze
-
-    BLACKLISTED_ERRORS = [].freeze
+    ERROR_HANDLER = lambda do |error|
+      classes = [
+        Interrupt,
+        NoMemoryError,
+        SignalException,
+        SystemExit
+      ]
+      raise error if classes.any? { |klass| error.is_a?(klass) }
+    end
```

Comment:
I do agree that the `true`/`false` meaning is not apparent. What about a passed in object that responds with `#handle_error(error)` to be called in the case where the user want it to affect stoplight.